Test/AccountTest.cs

```diff
@@ -29,7 +29,8 @@ public void CreateAccountWithParameters()
                 VatNumber = "my-vat-number",
                 TaxExempt = true,
                 EntityUseCode = "I",
-                CcEmails = "cc1@test.com,cc2@test.com"
+                CcEmails = "cc1@test.com,cc2@test.com",
+                Address = new Address()
```

Comment:
For reference: https://github.com/recurly/recurly-client-net/blob/master/Library/Address.cs#L15-L20